docs/notifications.md

```diff
@@ -132,7 +132,7 @@ docker run -d \
   --name watchtower \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -e WATCHTOWER_NOTIFICATIONS=shoutrrr \
-  -e WATCHTOWER_NOTIFICATION_SHOUTRRR_URL=discord://channel/token \
-  -e WATCHTOWER_NOTIFICATION_SHOUTRRR_URL=slack://watchtower@token-a/token-b/token-c \
+  -e WATCHTOWER_NOTIFICATION_URL=discord://channel/token \
+  -e WATCHTOWER_NOTIFICATION_URL=slack://watchtower@token-a/token-b/token-c \
```

Comment:
I think this will only result in one the URLs being set. One variable will override the other. I don't know how Viper does slices and environment variable (comma separated strings? space separated? ...?)

Reply:
Confirmed. I will try out the slicing approach.

Reply:
Hmm, I remember we did something around this in shoutrrr. I'll need to check it up though. From the top of my head, I think we decided on comma-separated URLs or something like that

Reply:
Space separation works. I’ll adjust the documentation.